### PR TITLE
Changed install task Docker Auto-Start to use Ansible module so sudo/…

### PR DESCRIPTION
**Pull request by amaurer**
Changing to use ansible service module.
Tags added which are ones I originally searched for
Removing dependency on pip and python setuptools

### CODE REVIEW

File: tasks/login.yml [hunk 1]
```diff
@@ -1,7 +1,4 @@
 ---
 - name: Login to docker hub
-  docker_login:
-    username: "{{ docker_pi_hub_username }}"
-    password: "{{ docker_pi_hub_password }}"
-    email: "{{ docker_pi_hub_email }}"
+  command: docker login --username="{{ docker_pi_hub_username }}" --password="{{ docker_pi_hub_password }}"
```
Comment (benaud12) [inline tasks/login.yml:3]: Since email isn't needed anymore could you also remove it from the main task file and default variables? And update the README?

**Review by benaud12 — COMMENTED** [review]
Hey Andrew - thanks for your improvements. Just one small change requested then I'll merge

File: tasks/login.yml (hunk 1 above)
Comment (amaurer) [reply tasks/login.yml:3]: Yep - oversight for sure.  Done.

**Review by benaud12 — APPROVED** [review]

File: tasks/login.yml (hunk 1 above)
Comment (benaud12) [reply tasks/login.yml:3]: Great - Thanks again! :)

**Pull request merged**